### DISCUSSION TITLE
fp16/xformers causes training to do nothing

### DISCUSSION THREAD

**Discussion by doodguymandood** [d8ahazard/sd_dreambooth_extension#868] [Q&A] [answered]
I'm sure this is just me being big dumb but yeah for whatever reason enabling this just makes training have no effect. is training steps or learning rate drastically different for these settings, or is it some other setting conflicting?

**Comment by David-337** [comment, marked as answer]
A while ago I stumbled into exactly this problem, turned out that the xformers wheel that I installed was corrupt, downloading and installing an xformers wheel from a different source fixed the problem for me.

Also note that as detailed [here](https://github.com/d8ahazard/sd_dreambooth_extension/wiki/Extremely-Experimental-Libs) currently the only supported version of xformers is 0.0.14.dev0, you may have a newer or older version installed which could also be causing problems.

Make sure that your xformers, as well as all of the other packages listed on that page, match their supported versions.

**Reply by doodguymandood** [reply]
ack ok it's my xformers most likely, any real difference between the new versions and the old outside of dreambooth?

**Reply by David-337** [reply]
From what I can see in the commits history of xformers there's quite a lot of bugfixes in the newer versions, however I compared generating images with 0.0.14.dev0 and 0.0.16 and found no noticeable difference in performance or quality.
